lib/common/logging.c

```diff
@@ -822,6 +822,25 @@ crm_log_preinit(const char *entity, int argc, char **argv)
             set_format_string(lpc, crm_system_name, pid, nodename);
         }
     }
+
+#ifdef PCMK__SUPPORT_NLS
```

Comment:
This should just be ENABLE_NLS.

Reply:
Aha, that's probably the issue

Reply:
rebased on current main, only change is this fix